docs/content.rst

```diff
@@ -593,7 +593,7 @@ the standard folder:
     class  MyFolder(SequentialAutoNamingFolder):
         """ Override Folder content type """
 
-The add view for Documents can then be edited to no longer require a name:
+The ``add view`` for Documents can then be edited to no longer require a name:
 
```

Comment:
Correct. When you reference that something can be edited, you need to provide the source. This "add view for Documents" appears from nowhere.